pkg/virt-launcher/notify-client/client.go

```diff
@@ -238,7 +238,26 @@ func newWatchEventError(err error) watch.Event {
 	return watch.Event{Type: watch.Error, Object: &metav1.Status{Status: metav1.StatusFailure, Message: err.Error()}}
 }
 
-func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEvent, client *Notifier, events chan watch.Event,
+type eventCaller struct {
+	domainStatus             api.LifeCycle
+	domainStatusChangeReason api.StateChangeReason
+}
+
+func (e *eventCaller) printStatus(status *api.DomainStatus) {
+	v := 2
+	if status.Status == e.domainStatus && status.Reason == e.domainStatusChangeReason {
+		// Status hasn't changed so log only in higher verbosity.
+		v = 3
```

Comment:
I like this mechanism but won't this fully overwrite the log verbosity set by the user in `developerConfiguration`? Maybe instead of hardcoding this 3 we can use the value set by the user instead. Just a thought.

Reply:
Mmh, not sure I completely get this. This should be equivalent to the original code where the output line was outputed when the level was 3 or higher. 
Maybe I should hardcod 2 but use the constant '2'

Reply:
Oops nevermind,  I fully misunderstood the logic here. Looks good to me then.

Reply:
No, I can't use `INFO` since we don't have mapping between the LogLevel and the integers